data_lang/pretty.py

```diff
@@ -76,6 +76,8 @@
 #
 # IfFlat(a, b) prints a if in flat mode or b otherwise.
 #
+# Flat(a) prints a in flat mode. You should generally not need to use it.
```

Comment:
Hm I guess this reminds me that  I still want to test the policy/mechanism split, with the ASDL pretty printer

I guess that would also use _Tabular, but it doesn't necessarily use IfFlat / Flat directly either

And there is a bit of dependency issue because we are maintaining a separation of ASDL from Oils

e.g. the `value_t -> doc_t` transformation is ideally separate, and then we also have an ASDL `hnode_t -> doc_t` transform